tests/SchemaTest.php

```diff
@@ -502,9 +504,51 @@ public function testGetIndexes()
         $this->assertSame([], $indexes);
     }
 
+    /** @todo requires SearchIndex support */
```

Comment:
Note this TODO item. I presume both this and `testVectorSearchIndex()` below require some runtime checks to determine if the methods can be executed successfully.